modules/cat/fastq/main.nf

```diff
@@ -35,8 +35,8 @@ process CAT_FASTQ {
             def read2 = []
             readList.eachWithIndex{ v, ix -> ( ix & 1 ? read2 : read1 ) << v }
             """
-            cat ${read1.sort().join(' ')} > ${prefix}_1.merged.fastq.gz
-            cat ${read2.sort().join(' ')} > ${prefix}_2.merged.fastq.gz
+            cat ${read1.join(' ')} > ${prefix}_1.merged.fastq.gz
+            cat ${read2.join(' ')} > ${prefix}_2.merged.fastq.gz
```

Comment:
is there any reason why the sort is not needed anymore? 
On most filesystems files appear by the order they were saved, not alphabetically.

Reply:
Ah, it's because we now need to organise the the files by the `input*` folder so they are concatenated in the correct order. This will now be dictated by the order in which they are supplied in the input channel and not by filename.

Reply:
ok, perfect!